GVFS/GVFS.Common/ProcessHelperImpl.cs

```diff
@@ -0,0 +1,16 @@
+namespace GVFS.Common
+{
+    /// <summary>
+    /// Default product implementation of IProcessHelper
+    /// interface. Delegates calls to static ProcessHelper class. This
+    /// class can be used to enable testing of components that call
+    /// into the ProcessHelper functionality.
+    /// </summary>
+    public class ProcessHelperImpl : IProcessHelper

```

Comment:
I would name this `ProcessRunner` or `ProcessLauncher` and if we add more to it rename it but since all it is doing is one thing right now we can use a specific name for the class.